client/blocks/eligibility-warnings/hold-list.jsx

```diff
@@ -15,81 +15,67 @@ import SectionHeader from 'components/section-header';
 
 // Mapping eligibility holds to messages that will be shown to the user
 // TODO: update supportUrls and maybe create similar mapping for warnings
-function getHoldMessages( translate ) {
+function getHoldMessages( siteSlug, translate ) {
 	return {
-		PLACEHOLDER: {
-			title: '',
-			description: '',
-			supportUrl: '',
-		},
 		TRANSFER_ALREADY_EXISTS: {
 			title: translate( 'Installation in progress' ),
-			description: translate( 'Another installation is already in progress.' ),
-			supportUrl: 'https://wordpress.com/help'
-		},
-		NO_BUSINESS_PLAN: {
-			title: translate( 'Business plan required' ),
-			description: translate( 'This feature is only allowed on sites with a business plan.' ),
-			supportUrl: 'https://support.wordpress.com/'
+			description: translate( 'Please wait for the other installation to complete, then try again.' ),
+			supportUrl: 'https://wordpress.com/help',
 		},
 		NO_JETPACK_SITES: {
-			title: translate( 'Jetpack site' ),
-			description: translate( 'This feature is not supported on Jetpack sites.' ),
-			supportUrl: 'https://wordpress.com/help'
+			title: translate( 'Jetpack not supported' ),
```

Comment:
Shouldn't it say "Jetpack site not supported" instead? We do the same for VIP sites (=> "VIP site not supported").

This looks like the Jetpack plugin is not supported, not Jetpack site.